neqo-transport/src/crypto.rs

```diff
@@ -1030,6 +1030,6 @@ impl Default for CryptoStreams {
 #[derive(Debug, Clone)]
 pub struct CryptoRecoveryToken {
     space: PNSpace,
-    offset: u64,
-    length: usize,
+    pub offset: u64,
+    pub length: usize,
```

Comment:
in qlog::packet_lost we need this to call `QuicFrame::crypto()`. I thought since CryptoRecoveryToken was pretty much a group of values that just making them pub was reasonable, but if you want accessor fns we can do that too.

Reply:
Now you made me look closer at that, I think that it is very badly wrong to be doing this.  See the comment about that above.